app/models/tv_actor_credit.rb

```diff
@@ -1,35 +1,32 @@
 class TVActorCredit
-  def initialize(show_name, show_id, actor_name, actor_id, character, seasons, episodes)
+  attr_accessor :show_name, :show_id, :actor_name, :actor_id, :character, :seasons, :episodes, :profile_path, :known_for
 
-    @show_name = show_name
-    @show_id = show_id
-    @actor_name = actor_name
+  def initialize(show_name:, show_id:, actor_name:, actor_id:, character:, seasons:, episodes:, profile_path:, known_for:)
     @actor_id = actor_id
+    @actor_name = actor_name
     @character = character
-    @seasons = seasons
     @episodes = episodes
+    @known_for = known_for
+    @profile_path = profile_path
+    @show_id = show_id
+    @show_name = show_name
+    @seasons = seasons
+  end
 
-  end #init
-
-  attr_accessor :show_name, :show_id, :actor_name, :actor_id, :character, :seasons, :episodes
-
-  def self.parse_results(result)
-    @show_name = result[:media][:name]
-    @show_id = result[:media][:id]
-    @actor_name = result[:person][:name]
-    @actor_id = result[:person][:id]
-    @character = result[:media][:character]
```

Comment:
I've reconciled and removed these instance variables. They're accessible only as attributes in this object now.